dev/devicelab/manifest.yaml

```diff
@@ -396,13 +396,15 @@ tasks:
     description: >
       Measures the startup time of the Flutter Gallery app on 32-bit iOS (iPhone 4S).
     stage: devicelab_ios
+    flaky: true # Currently broken due to infra outage.
```

Comment:
Can this link to something that tracks the outage? Ideally an issue that gets closed when the outage is resolved so we know when to re-enable this?

Reply:
I asked in the chat, waiting to hear back